packages/traceability/issuer-irec-api/src/pods/certification-request/handlers/get-all-certification-requests.handler.ts

```diff
@@ -0,0 +1,39 @@
+import { IQueryHandler, QueryHandler } from '@nestjs/cqrs';
+import { InjectRepository } from '@nestjs/typeorm';
+import { In, Repository } from 'typeorm';
+
+import { CertificationRequest, GetAllCertificationRequestsQuery } from '@energyweb/issuer-api';
+import { FullCertificationRequestDTO } from '../full-certification-request.dto';
+import { IrecCertificationRequest } from '../irec-certification-request.entity';
+
+@QueryHandler(GetAllCertificationRequestsQuery)
+export class GetAllCertificationRequestsHandler
+    implements IQueryHandler<GetAllCertificationRequestsQuery>
+{
+    constructor(
+        @InjectRepository(CertificationRequest)
+        private readonly repository: Repository<CertificationRequest>,
+        @InjectRepository(IrecCertificationRequest)
+        private readonly irecRepository: Repository<IrecCertificationRequest>
+    ) {}
+
+    async execute({
+        query
+    }: GetAllCertificationRequestsQuery): Promise<FullCertificationRequestDTO[]> {
+        const certificationRequests = await this.repository.find(query);
```

Comment:
Can't we call `super()` here as well to extend the original handler from issuer-api?

Reply:
done